client/src/components/styles/introduction.css

```diff
@@ -18,14 +19,14 @@
 }
 
 .text_title {
-  color: darkslategray;
+  color: rgb(72, 73, 75);
   font-size: 12px;
   margin-left: 10%;
   margin-bottom: 3%
```

Comment:
semi-colon. 

Also not a critique (because I don't know exactly what you're trying to achieve here), more of an initial concern with the hope that you can explain why your way is right:
Setting margins or positioning elements with a percentage like this frightens me because it is technically unpredictable. Could this not be handled with the more trustworthy/absolute `px`? Interested to hear what's going on with this. I see you're using it frequently in your styles but this is the only place I'll mention it.

Reply:
I used %s because if you look at airbnb's website, the content/pictures are always relative to the size of the browser whether it's full screen, half screen, etc. I just wanted mine to have the same effect where the images are proportional to the screen size rather than cut off when you make the browser smaller.